src/datasets/non_desktop/events_daily/reference.md

```diff
@@ -0,0 +1,112 @@
+# Events Daily
+
+<!-- toc -->
+
+# Introduction
+
+`org_mozilla_firefox.events_daily` is designed to answer questions about events. These include:
+- Funnels
+- Event Counts
+- User Flows
+
+# Contents
+
+`events_daily` has one row per-client per-day, much the same as `clients_daily`. The table is created in a two-step process:
+
+1. An ancillary table, `event_types`, is updated with the new events seen on that day. Each event is mapped to a unique
+    unicode character, and each event property (the `extras` fields) are also mapped to a unique unicode character.
+2. For every user, that day's events are mapped to their associated unicode characters (including `event_properties`).
+    The strings are aggregated and comma-separated, giving a single ordered string that represents all of that user's
+    events on that day.
+
+For Fenix, we aggregate the events ping data _only_. If you're looking for events in other pings, you'll need to query them directly.
+
+Included in this data is a set of dimensional information about the user, also derived from the events ping. The full list of fields is available [in the query.](https://github.com/mozilla/bigquery-etl/blob/master/sql/org_mozilla_firefox_derived/events_daily_v1/query.sql#L48).
+
+## Limitations
+
+This approach makes some queries fast and easy, but has some limits:
+1. Each product is limited to at most 1 Million unique event types
+2. Each event property is limited to at most 1 Million values. As a result, [some Fenix event properties are not included in this table](https://github.com/mozilla/bigquery-etl/blob/master/sql/org_mozilla_firefox_derived/event_types_v1/query.sql#L89).
+3. Queries do not know the amount of time that passed between events, only that they occurred on the same day
+        *Note*: This can be alleviated by sessionizing and splitting the events string using a session_start event.
+            For Fenix this could be [`events.app_opened_all_startup`](https://github.com/mozilla-mobile/fenix/blob/master/app/metrics.yaml#L11).
+
+# Accessing the Data
+
+While it is possible to build queries that access this events data directly, the Data Platform instead recommends using a set of stored procedures we have available.
+These procedures create easily queryable views that hide the complexity of the event representation. The [`mozfun` library documentation](https://mozilla.github.io/bigquery-etl/mozfun/event_analysis/)
```

Comment:
Generally best practice to avoid using "easily" in documentation. Simplifying the sentence also removes the need to add "queryable" to the dictionary. :)

```suggestion
These procedures create BigQuery views that hide the complexity of the event representation. The [`mozfun` library documentation](https://mozilla.github.io/bigquery-etl/mozfun/event_analysis/)
```